Build/Scripts/runTests.sh

```diff
@@ -568,7 +568,7 @@ case ${TEST_SUITE} in
         SUITE_EXIT_CODE=$?
         ;;
     phpstanGenerateBaseline)
-        COMMAND="composer phpstan:baseline"
+        COMMAND="composer phpstan:baseline --allow-empty-baseline"
```

Comment:
So this should be kept as the following?
```suggestion
        COMMAND="composer phpstan:baseline"
```